python/foglamp/services/core/api/backup_restore.py

```diff
@@ -7,113 +7,148 @@
 """Backup and Restore Rest API support"""
 
 from aiohttp import web
+from enum import IntEnum
+from collections import OrderedDict
+
+from foglamp.services.core import connect
+from foglamp.plugins.storage.postgres.backup_restore.backup_postgres import Backup
+from foglamp.plugins.storage.postgres.backup_restore import exceptions
+
 # TODO: remove this and call actual class methods
 from unittest.mock import MagicMock
-Backup = MagicMock()
+Backup_Mock = MagicMock()
```

Comment:
Clearly we can not merge with this still in place

Reply:
Ok. We can do it either way:
1. Pull FOGL-861 and FOGL-834 in current sprint and implement the mocked methods in this pull only.
2. Remove routes for restore and subsequently the restore api code with a TODO to be addressed in a future sprint. That way we'll have SOME logic for restore stuff but not accessible by api.

Reply:
We do not have the bandwidth to pull these other tickets into the sprint, Let's pull the route out and push with the mocks. I don't like the idea of having mocked things in the repository.

Reply:
Mocks removed. calling the restore endpoint raises exception "Restore backup method is not implemented yet"